scss/bootstrap/variants/_outlined-buttons.scss

```diff
@@ -0,0 +1,41 @@
+.btn-default-outline {
+  @include button-outline-variant($crm-white);
+  color: $crm-white !important;
+  border-color: $crm-white !important;
+
+  .open &,
+  &:hover,
+  &:active {
+    background-color: $crm-white !important;
+    color: $crm-dark-blue !important;
+  }
+}
+
+.btn-primary-outline {
+  @include button-outline-variant($btn-primary-bg);
```

Comment:
But it works and and doesn't throw any error. Mixins are imported just before with `@import "mixins/*";`